### ISSUE TITLE
Shared cursors on Dashboard Cells

### ISSUE THREAD

**Issue by drdelambre** [influxdata/influxdb#16612]
Chronograf 1.x used to share the cursor pointers across all cells on a dashboard, we should bring parity to 2.0. It looks a little something like this:

![Jan-21-2020 16-11-47](https://user-images.githubusercontent.com/1434802/72853772-3bc0a380-3c67-11ea-8749-a8b700caa2e7.gif)

**Comment by drdelambre** [comment]
added context from Russ:

i was playing around with another popular dashboarding tool. So zooming in on a single graph should adjust the start and stop time range of a dashboard. Any query using those start and stop variables would then be refreshed with the new range. For cells that are using a fixed time range, the “other dashboarding tool” zooms and refreshed the query, even if it is fixed.

that last part is open for interpretation IMO, but the nice thing that do is say is that there is data outside of the time range, which would make it obvious

**Comment by rawkode** [comment]
Was about to log this issue. Lets make this so, please :smile:

**Comment by philjb** [comment]
I support this.

**Comment by NonLogicalDev** [comment]
I feel like dashboard is pretty useless without this feature, as there is no way to visually correlate metrics =[. I am a quite surprised that it is missing in V2. I hope this is just because of prioritization to get the V2 out the door.

**Comment by kamermans** [comment]
I agree with @NonLogicalDev and the rest, this feature is extremely important and works great in Chronograf 1.x.  It allows you to have graphs of different widths and still correlate times between them.  I am holding back our company from upgrading to 2.x until this feature is available (and until we can translate all of our dashboards to Flux 😐).

**Comment by NonLogicalDev** [comment]
Hey Influx team, just wanted to bump this thread.

I saw that annotations are being worked on, though they have not landed yet in OSS version (#21591). Is adding shared cursors on the roadmap?

**Comment by NonLogicalDev** [comment]
Bumping again, in case there are any updates. Just wondering if this feature is anywhere on the radar/roadmap.

**Comment by NonLogicalDev** [comment]
Seems like there is another issue lurking in the same vein and probably in need of the same underlying synchronisation mechanism:
https://github.com/influxdata/influxdb/issues/19962

Would be very nice to see movement on either one of those probably even more so on the zoom one. 

It is very nice to see annotations implemented but again those are also not shared between cells.

**Comment by NonLogicalDev** [comment]
Bumping the issue to keep it active, would be very nice to see this feature someday.

**Comment by daubman** [comment]
Since it's been over a year, and I just came across this as a new user searching for exactly this type of tooling to correlate and visualize across cells, figured I'd give this another bump (I was certain it must be possible and I just hadn't figured out how)
tl;dr another year, another request for this 🙏🏻
